frontend/src/pages/modelServing/screens/projects/ServingRuntimeModal/ManageServingRuntimeModal.tsx

```diff
@@ -1,25 +1,22 @@
 import * as React from 'react';
+import { AxiosError } from 'axios';
 import {
   Alert,
-  Button,
```

Comment:
Ok, I'll add the comment here, I think I explained myself here wrong yesterday, I do apologize, we still need the accessReviewResource and all of his logic, the only thing we needed to do was remove it from the serving runtime conditional

Reply:
Ok, sorry for the misunderstanding. I thought the issue was that the `useAccessReview` itself wouldn't work properly for the restricted user.